go-controller/pkg/ovn/obj_retry.go

```diff
@@ -0,0 +1,128 @@
+package ovn
+
+import (
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+type retryObjEntry struct {
+	newObj     interface{}
```

Comment:
is there ever a case when both newObj and oldObj are set?
If not, could this just be a obj and a `bool isDelete` ?
Please forgive and ignore me if thatis a silly idea. ;)

Reply:
during update ? u can have old and new objects that need to be taken care of see the logic in the watch code path in ovn.go

Reply:
ah. ack. thanks for the clarification.